generate_ballots.py

```diff
@@ -0,0 +1,45 @@
+import yaml
```

Comment:
Before merging, can you resolve the merge conflicts?

Reply:
I just removed the conflicts I think